src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -2471,6 +2471,12 @@ well as menu structures (for main menu and popup menus).
         windowMode="main"
         desc="Clear console"/>
 
+   <cmd id="clearBuild"
+        buttonLabel=""
+        label="Clear Build"
+        windowMode="main"
+        desc="Clear build"/>
```

Comment:
Suggest adding `context="packageDevelopment"` to this command to match other build-pane related commands. This controls grouping in the keyboard shortcut reference panel, and appears in the command-palette.